account_move_line_accounting_description_sale/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Acccount Move Accounting Sale Order Description",
+    "version": "14.0.1.0.0",
+    "summary": "Adds an 'accounting description' on sale order",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "license": "AGPL-3",
+    "category": "Accounting & Finance",
+    "depends": ["sale", "product", "account_move_line_accounting_description"],
+    "data": [],
```

Comment:
You could remove empty key and set module as auto install. (also for the glue to purchase module)